tools/find_image.rb

```diff
@@ -30,7 +30,7 @@ def run
       c.action do |args, options|
         options.default \
           :build_type => "nightly-",
-          :url => "https://openshift-release.svc.ci.openshift.org/"
+          :url => "https://openshift-release.apps.ci.l2s4.p1.openshiftapps.com/"
```

Comment:
@lihongyan1 this file change doesn't seem to belong here but in a separate PR.  Also, please don't change the default url...the script should follow the url direct.